.circleci/config.yml

```diff
@@ -598,7 +606,6 @@ jobs:
       - store_artifacts:
           path: fastlane/test_output
           destination: scan-test-output
-      - slack-notify-on-fail
```

Comment:
If I removed it is because it's only invoked in `build-test` and that runs on PRs